telegram/ext/_application.py

```diff
@@ -1399,8 +1429,9 @@ async def update_persistence(self) -> None:
         along with :attr:`~telegram.ext.ExtBot.callback_data_cache` and the conversation states of
         any persistent :class:`~telegram.ext.ConversationHandler` registered for this application.
 
-        For :attr:`user_data`, :attr:`chat_data`, only entries used since the last run of this
-        method are updated.
+        For :attr:`user_data` and :attr:`chat_data`, only those entries are updated that were used
+        or manually marked via :meth:`mark_data_for_update_persistence` since the last run of this
+        method.
```

Comment:
```suggestion
        For :attr:`user_data` and :attr:`chat_data`, only the entries are updated which either were 
        used or have been manually marked via :meth:`mark_data_for_update_persistence` since 
        the last run of this method.
```